docs/Change_Log.md

```diff
@@ -1,5 +1,17 @@
 # Change Log
 
+## Changes to Slash Command Response Types and Flags
+
+## February XX, 2021
+
+Changes to interaction response types have been made to support better designs for Slash Commands:
+
+- Type `2` `Acknowledge` has been deprecated
+- Type `3` `ChannelMessage` has been deprecated
+- Type `5` `AcknowledgeWithSource` has been renamed to `DeferredChannelMessageWithSource`
```

Comment:
Given everything has a source message now, should the `WithSource` suffix be removed? (it's extra clutter... Though the entire file about slash commands reads like Java class names...)